images/extension-builders/tinygo/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Copyright 2020 Tetrate
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+# Builder image for Envoy Wasm extensions written in Go.
+#
+
+FROM tinygo/tinygo:0.17.0
+
+ENV GOCACHE=/source/build/.gocache
```

Comment:
Can we include `.gitignore` file into the set of files generated by `getenvoy extension init` ?

E.g., with the following contents:
```
/build/
```

We do that for `Rust` extensions.